### ISSUE TITLE
GGA and more DFT functionals in FDMNES

### ISSUE THREAD

**Issue by chrinide** [gudasergey/fdmnes#9]
Dear FDMNES developers,

  Is there any plan to support GGA functional like PBE/BLYP and further more  DFT functionals in FDMNES?

**Comment by gudasergey** [comment]
Yes, it is. Yves plans to put better functionals in the future. He cannot give when for the moment because he is completely involved in other progress in the code. The nearest goal is to code RIXS.
